guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,577 @@
+      alt: A mobile wallet interface showing a mint details screen with an input field to edit the mint url.
+      caption: Give the user the option to edit the mint url.
+    - file: cashu-mint-url-edit
+      modalImage: cashu-mint-url-edit@2x
+      alt: A mobile wallet interface showing a user typing a mint url into the mint url input field.
+      caption: Allow the user to manually type the mint url.
+
+images_p2pk:
+    - file: generate-token
+      modalImage: generate-token@2x
+      alt: A mobile wallet interface showing a payment about to be sent. The bitcoin and fiat denominated balance is displayed at the top, along with an advance options button at the bottom.
+      caption: P2PK is an advance feature and should not be displayed as a regular feature.
+    - file: advance-settings
+      modalImage: advance-settings@2x
+      alt: A mobile wallet interface with a bottom sheet that lists the following actions, Coin control, and P2PK Lock.
+      caption: Consider showing advance features like P2PK in a bottom sheet or modal.
+    - file: enter-p2pk
+      modalImage: enter-p2pk@2x
+      alt: A mobile wallet interface with an input field to enter the recipient's public key.
+      caption: Allow the user to manually enter the recipient's public key or 
+    - file: share-ecash
+      modalImage: share-ecash@2x
+      alt: A QR code for a generate ecash token with the token string available to copy to clipboard.
+      caption: Allow the user to copy the token string to clipboard by tapping on QR code or text string.
+
+images_guardian-status:
+    - file: federation-guardian-good
+      modalImage: federation-guardian-good@2x
+      alt: A mobile wallet interface with a screen that shows a federation and their health status in the form of a green progress bar. The more the progress bar is filled, the more the guardian has maintained uptime.
+      caption: Consider the privacy implications of showing the status and name of guardians.
+    - file: federation-guardian-ok
+      modalImage: federation-guardian-ok@2x
+      alt: A mobile wallet interface with a screen that shows a federation experiencing partial connecetivity issues due to one guardian being offline.
+      caption: Show the user the current status of the federation.
+    - file: federation-guardian-bad
+      modalImage: federation-guardian-bad@2x
+      alt: A mobile wallet interface with a screen that shows an unreliable federation and their health status in the form of a red progress bar along with a message that the federation is offline.
+      caption: Clearly inform a user if a federation is offline.
+
+images_guardian-expiry:
+    - file: federation-guardian-expiry
+      modalImage: federation-guardian-expiry@2x
+      alt: A mobile wallet interface with a screen that shows a countdown timer and a message that the federation will shut down in 15 days.
+      caption: Federation expiration timestamp displayed as a countdown alter when less 30 days remaining.
```

Comment:
Nit: "alter" -> "alert"

Also it would be useful for federations to be able to extend their expiration time (i.e. they're promising to not shut down before a certain date, but not promising that they will _definitely_ shut down at that date. They might tack on more time later)